config/schema/elasticsearch_types/cma_case.json

```diff
@@ -148,6 +156,10 @@
         "label": "Motor industry",
         "value": "motor-industry"
       },
+      {
+        "label": "Not applicable",
+        "value": "not-applicable"
+      },
```

Comment:
It seems really strange that this is the way that CMA have decided to go, and that they are willing to assign all new cases to a "not applicable" market sector forever more, rather than having market sector be an optional field.

Was that considered as an option?

Reply:
It was considered, and I tried it quickly. What I found was that with the multi-select input component in specialist publisher, if you made it optional the value when no options were selected was an array with an empty string. I could have written a transformer for this case that handled the `[""]` by removing the key or providing a nil value, but that felt pretty hacky to me, and for all I know that might be other finders where `[""]` is a valid value. So we agreed with the CMA to take this approach.